src/main/java/jwt/JwtUtils.java

```diff
@@ -0,0 +1,48 @@
+package jwt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.beans.factory.annotation.Value;
+
+import javax.crypto.spec.SecretKeySpec;
+import java.security.Key;
+import java.util.Date;
+
+public class JwtUtils {
+    private static final long EXPIRATION_TIME = 86400000;
```

Comment:
상수 분리 👍👍 86400000은 하루를 밀리초로 표현한 것 같아요. 다음은 제안사항인데 혜빈님의 의견이 궁금해요! 

**86400000의 단위가 밀리초라는 걸 표현하면 어떨까요?** 

```suggestion
    private static final long EXPIRATION_TIME_MILLIS = 86400000;
```

**86400000가 하루라는 걸 표현하면 어떨까요?** 

```suggestion
    private static final long EXPIRATION_TIME_MILLIS = 1000 * 60 * 60 * 24;
```

Reply:
오! 굉장히 좋은 생각인 거 같아요!!
저는 86400000라는 숫자가 밀리초를 의미한다는 걸 알지만 코드를 처음 보는 사람들은 헷갈릴 수 있겠다는 생각이 드네요!!
``` java
EXPIRATION_TIME_MILLIS = 1000 * 60 * 60 * 24;
```
확실히 이렇게 표시해주는 게 개발자 친화적인 표기법인 거 같습니다 :)
좋은 제안 감사합니다!! 망초님 조언대로 코드 변경하였습니다 *v*